server/manager/src/main/java/org/apache/accumulo/manager/tableOps/compact/CompactionDriver.java

```diff
@@ -129,7 +129,8 @@ public long isReady(FateId fateId, Manager manager) throws Exception {
     return sleepTime;
   }
 
-  private boolean isCancelled(FateId fateId, ServerContext context)
+  // visible for testing
+  protected boolean isCancelled(FateId fateId, ServerContext context)
```

Comment:
```suggestion
  // visible for testing
  @VisibleForTesting
  protected boolean isCancelled(FateId fateId, ServerContext context)
```
Could add this annotation here